myworkspacestaff_managementlog.txt

```diff
@@ -0,0 +1,355 @@
+2020-05-17 20:30:55.654  INFO 8220 --- [restartedMain] c.s.StaffManagementApplication           : Starting StaffManagementApplication on DESKTOP-SLLSFPL with PID 8220 (E:\myworkspace\staff_management\staff-management-api\target\classes started by poonam.sharma in E:\myworkspace\staff_management\staff-management-api)
```

Comment:
remove this file. i would suggest to add this in `.gitignore` after deleting from repo